src/bikeracks/votes.py

```diff
@@ -111,5 +111,53 @@ def submit_vote():
     # maybe return 500 here
     return "", 500
     
-
+@votes.route('/unvote', methods=['POST'])
+def unvote():
+    # remove a vote that the user had previously made
+    if request.method == 'POST':
+        try:
+            rack_id = request.args.get('rack_id') or None
+            user_id = request.args.get('user_id') or None
+            vote_type = request.args.get('vote_type') or None
+            
+            # connect to db
+            db = get_db()
+            
+            # remove user's vote for this rack from votes table and update bikeracks vote count
+            
+            #remove user's vote 
+            query = """
+                    DELETE FROM
+                        votes
+                    WHERE
+                        user_id=? AND
+                        vote_type=? AND
+                        rack_id=?
+                    """
+            db.execute(query, (user_id, vote_type, rack_id))
+            
+            # decrement the vote count in the bikeracks row for this rack
+            if int(vote_type) == 1:
+                vote_count = "upvote_count"
```

Comment:
Make it clear that `vote_count` is a column name.

Reply:
Changed it to count_column_name